src/components/Table/TableRow/TableRow.scss

```diff
@@ -0,0 +1,21 @@
+.moonstone-table_row {
+    border-bottom: 1px solid var(--color-gray40);
+
+    &:hover {
+        background-color: var(--color-gray_light40);
+    }
+
+    &.moonstone-table_row-selected {
```

Comment:
This comment is for all CSS class names. I propose to use the same name as the component but in camelCase and use underscore for modifiers like: `moonstone-tableRow_selected`. 
I know we don't have a real naming convention for CSS (yet). What do you think?

Reply:
Same comment as above...I think this should be `moonstone-table_row-selected`.